### PR TITLE
[skip ci] Fix spells in documentation

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

File: lib/active_admin/dsl.rb [hunk 1]
```diff
@@ -128,7 +128,7 @@ def menu(options = {})
       config.menu_item_options = options
     end
 
-    # Set the name of the navigation menu to display. This is mainly used in conjuction with the
+    # Set the name of the navigation menu to display. This is mainly used in conjunction with the
```
Comment (davydovanton) [inline lib/active_admin/dsl.rb]: :confused:

Comment (timoschilling) [inline lib/active_admin/dsl.rb]: don't worry, your change is ok

**Comment by timoschilling** [comment]
Please put the `[skip ci]` at the end of the commit name, next time

**Comment by davydovanton** [comment]
@timoschilling okay, thanks! :star2:

**Pull request merged**